### ISSUE TITLE
Transfer project to github.com/sugarlabs

### ISSUE THREAD

**Issue by davelab6** [sugarlabs-infra/translation-team#16]
I propose to transfer this project to github.com/sugarlabs

Does anyone object?

**Comment by codewiz** [comment]
I do not object, but let's wait to hear back from icarito and cjl. They are the translation team after all..

**Comment by codewiz** [comment]
@icarito @leonardcj see my previous comment.

**Comment by davelab6** [comment]
You can see the list of people who get emails for all issues/PRs at https://github.com/sugarlabs-infra/translation-team/watchers which includes Seb and Chris (and Sam C and Sam P :)

**Comment by icarito** [comment]
Yes I am in favour of having a public tracker for translation team. I am
also in favour of having a translation team.

**Comment by davelab6** [comment]
@leonardcj please let us know :)
